### DISCUSSION TITLE
Can Audio Alarm accompany a scheduled task?

### DISCUSSION THREAD

**Discussion by rajibando** [orgzly-revived/orgzly-android-revived#437] [Q&A]
May please this post be perused: 
[Need for a Reminder/Timer/Scheduler/Organiser which could be repeated as hourly, weekly, monthly, yearly, etc., reminders too, with audio Alarm](https://forum.f-droid.org/t/need-for-a-reminder-timer-scheduler-organiser-which-could-be-repeated-as-hourly-weekly-monthly-yearly-etc-reminders-too-with-audio-alarm/29608)

My last Question remains unanswered. It is a little long. So the silence could be justified. However, this question is short. So would expect a reply.

Please charge a price for your product. I will pay the declared price. I am little skeptical about donations. Accountability and service can be better ensured by cost and accompanying legal framework.

**Comment by amberin** [comment]
Just like with the other apps discussed in your linked page, Orgzly sends notifications for scheduled tasks. These notifications can be customized per app in Android, and by default they make the regular notification sound.

Does that answer your question?

**Reply by rajibando** [reply]
Thank you, Mr. [Andreasson](https://github.com/amberin), for your post. I am now a bit assured that I can hope to get your attention.

Yes, I see a notification for the tasks that I have set up, but I am yet to hear an alarm-like non-stop snooze-able sound to get my continued attention to the brink of my annoyance against the alarm. I am not joking. I can control that annoyance, if the task is that important to be performed on time. 

Hence the question if it isn't obvious: _How do I set up the app and the system to create a continuous alarm notification to the brink of disturbance, only to be snoozed, but not stopped until the task is complete?_

Please also attend the questions raised on my other post, [https://github.com/orgzly-revived/orgzly-android-revived/discussions/431](https://github.com/orgzly-revived/orgzly-android-revived/discussions/431), one along the body, the rest, towards the end.

Please mention a price for your product. I am ready to pay a price and assure the support and accountability of the Team App Developer. Business transactions are backed up by appropriate laws that try to ensure accountability and fairness.

I am skeptical of the donations model, as it doesn't ensure service availability for a product.
